torch/lib/c10d/TCPStore.cpp

```diff
@@ -104,13 +104,6 @@ void TCPStoreDaemon::run() {
         continue;
       }
 
-      if (fds[fdIdx].revents ^ POLLIN) {
```

Comment:
If `POLLIN` is a single bit, why are we using `^` instead of `&` here?

Reply:
To check if any bits other than POLLIN were set, which would be unexpected.

But on macOS I found POLLHUP to be set as well if the client closed its connection.